ipwb/replay.py

```diff
@@ -540,9 +553,6 @@ def generateCDXJTimeMapFromCDXJLines(cdxjLines, original, tmself, tgURI):
 
 # Fixes Flask issue of clipping queryString
 def getCompleteURI(uri):
```

Comment:
Why is this `getCompleteURI` function which is now converted to a  do nothing type function left here if you were to define a replacement function `compile_target_uri`?

Reply:
Oops, I forgot to remove it. Done now.